db/migrate/20200421141934_add_source_updated_at.rb

```diff
@@ -0,0 +1,5 @@
+class AddSourceUpdatedAt < ActiveRecord::Migration[5.2]
+  def change
+        add_column :volumes, :source_updated_at, :timestamp
```

Comment:
@pkomanek Please indent 2 spaces only.

Reply:
@pkomanek is this story only about adding the migration or how the field will get updated? Will `:source_updated_at` get touched without some modification in the model in some callback?

Reply:
@syncrou the migration is enough on the persister side. 
Columns for updating from payload are determined automatically by the InventoryCollectionBuilder
https://github.com/ManageIQ/inventory_refresh/blob/7accb70f1b2ae5befb3e2c2462e53993530db022/lib/inventory_refresh/inventory_collection/builder.rb#L219